src/vs/workbench/contrib/mcp/browser/mcpResourceQuickAccess.ts

```diff
@@ -205,6 +206,9 @@ export class McpResourcePickHelper {
 				const item = input.selectedItems[0];
 				if (item.id === currentID) {
 					resolve(input.value);
+				} else if (variable.explodable && item.label.endsWith('/') && item.label !== input.value) {
```

Comment:
Do you need to support `\` too?

Reply:
No -- these are completions in URI templates, and URIs only use `/`